appservice/src/SiteWrapper.ts

```diff
@@ -116,6 +120,67 @@ export class SiteWrapper {
         this.log(outputChannel, 'Deployment completed.');
     }
 
+    public async localGitDeploy(fsPath: string, client: WebSiteManagementClient, outputChannel: vscode.OutputChannel, servicePlan: string): Promise<void> {
+        vscode.window.showInformationMessage('It updated to 2.2');
```

Comment:
what's this message about?

Reply:
Whoops, sorry, that was just for testing. 😊